src/script.h

```diff
@@ -77,10 +77,11 @@ struct scriptRunCtx {
 };
 
 /* Scripts flags */
-#define SCRIPT_FLAG_NO_WRITES (1ULL<<0)
-#define SCRIPT_FLAG_ALLOW_OOM (1ULL<<1)
-#define SCRIPT_FLAG_ALLOW_STALE (1ULL<<3)
-#define SCRIPT_FLAG_NO_CLUSTER (1ULL<<4)
+#define SCRIPT_FLAG_IGNORE_FLAGS (1ULL<<0) /* Script backwards compatibility mode, no flags, use old defaults */
```

Comment:
any reason not to put that flag last (instead of changing so many lines)?
```suggestion
#define SCRIPT_FLAG_IGNORE_FLAGS (1ULL<<0) /* EVAL Script backwards compatible behavior, no shebang provided */
```